xrpl/models/utils.py

```diff
@@ -1,10 +1,13 @@
 """Helper util functions for the models module."""
 
-from dataclasses import is_dataclass
+from dataclasses import dataclass, is_dataclass
 from typing import Any, Dict, List, Type, TypeVar, cast
 
 from xrpl.models.exceptions import XRPLModelException
 
+# Python 3.10 and higer versions of Python enable a new KW_ONLY parameter in dataclass
```

Comment:
Two things:
1. Can you add a bit more detail on what this is accomplishing? 

Ex. That it allows IDEs to enable auto-complete, a link to the docs, etc. :)

(Likely the next person to touch this code won't have any context on what this line means)

2. Also, can you add in the comment where to find tests that this works? (Since the next person probably won't be able to independently discover those tests in 2 specific models :) )

Reply:
I'm not a big fan of specifying unit test file-paths in comments. If the said files move/update names in the future, it creates a downstream responsibility. But I don't know of any better way to link the unit test files

Reply:
Agreed: https://github.com/XRPLF/xrpl-py/pull/673#discussion_r1488524075
There's already a file for it actually: https://github.com/XRPLF/xrpl-py/blob/main/tests/unit/models/test_utils.py

Reply:
I have moved the unit tests into test_utils.py in [0cbc07e](https://github.com/XRPLF/xrpl-py/pull/673/commits/0cbc07eca9a5e647bf1be57c4e4403c888edf230)